spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -1,369 +1,372 @@
+  * [ ] Test Coverage is comprehensive
+* IF spell interacts with ChainLog
+  * [ ] ChainLog version is incremented based on update type
+    * Major -> New Vat (++.0.0)
+    * Minor -> Core Module (DSS) Update (e.g. Flapper) (0.++.0)
+    * Patch -> Collateral addition or addition/modification (0.0.++)
+  * [ ] New addresses are added to the `addresses_mainnet.sol`
+  * [ ] Changes are tested via `testNewOrUpdatedChainlogValues`
 * [ ] Ensure every spell variable is declared as `public`/`internal`
-* [ ] Ensure `immutable` visibility is only used when fetching addresses from the `ChainLog` via `DssExecLib.getChangelogAddress` and `constant` is used instead for static addresses
+* [ ] Ensure `immutable` visibility is only used when fetching addresses from the `ChainLog` via `DssExecLib.getChangelogAddress(key)` and `constant` is used instead for static addresses
   * [ ] Fetch addresses as type `address` and wrap with `Like` suffix interfaces inline (when making calls) unless archive patterns permit otherwise (Such as `MKR`)
   * [ ] Use the [DssExecLib Core Address Helpers](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L166) where possible (e.g. `DssExecLib.vat()`)
   * [ ] Where addresses are fetched from the `ChainLog`, the variable name must match the value of the ChainLog key for that address (e.g. `MCD_VAT` rather than `vat`), except where the archive pattern differs from this pattern (e.g. MKR)
 * [ ] Spell actions match the corresponding Exec Doc
-* [ ] Tests
+* Tests
   * [ ] Ensure each spell action has sufficient test coverage
     _List actions for which coverage was checked here_
   * [ ] Ensure every test function is declared as public if enabled or private if disabled
   * [ ] Ensure that the `DssExecLib.address` file is not being modified by the spell PR
   * [ ] Check all CI tests are passing as at the latest commit
     _Insert most recent commit hash where CI was passing_
   * [ ] Check all tests are passing locally using `make test`
-    * [ ] Ensure that only `ETH_RPC_URL` is being used from env (i.e. no `match`, `block` or similar are active in your env)
+  * [ ] Ensure that only `ETH_RPC_URL` is being used from env (i.e. no `match`, `block` or similar are active in your env)
 
-```
-_Insert your passing local tests here_
+```bash
+_Insert your local test logs here_
 ```
 
 ## Deployed Stage
 
-* [ ] Deployed Spell is Verified
-  * [ ] Optimization Enabled: No
-  * [ ] Other Settings: default evmVersion, GNU AGPLv3 license
-* [ ] Deployed Spell Code matches GitHub
-  * [ ] diffcheck etherscan source against spell PR (via `make diff-deployed-spell`)
-* [ ] Deployed Spell Etherscan Checks
-  * [ ] automated checks via `make check-deployed-spell`
-    * [ ] verified
-    * [ ] license type matches
-    * [ ] solc version matches
-    * [ ] optimizations are disabled
+* Source code settings
+  * [ ] Deployed spell is verified on etherscan
+  * [ ] Optimization enabled: No
```

Comment:
There were occasions in the past where the contract size limit required us to enable optimizations:

```suggestion
  * [ ] Optimization enabled: No UNLESS having issues with contract size AND all mitigation strategies (i.e.: removing revert strings) have failed.
```

Reply:
That's again improvement of the rule, not structure, but will take it as exception